source/appModules/hxmail.py

```diff
@@ -0,0 +1,35 @@
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2016 NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+

```

Comment:
Please add a docstring noting that this is an app module for the Windows 10 Mail app, since that isn't obvious from the executable name.